resources/css/site.css

```diff
@@ -964,21 +964,35 @@ a.text-center.btn.btn-outline-light.btn__book {
     }
 }
 
+/*
+    Fitz small improvements list - 23/11/2023
+    1. - .bg-gray classes removed in templates - background colour set to white.
+    2. - .shout.leads font-size set to 4.1rem
+    3. - .navbar .container-fluid max-width set to 1320px
+    4. - .fitz-footer max-width set to 1680px and horizontal scroll removed
+
+*/
+
+.col-12 {
+    max-width: 800px;
```

Comment:
I'm guessing this has been done to reduce the max width of the main content area to 800px wide.

Couple of things:

1. Generally we would recommend using REMs or EMs over px units, as they're better for accessibility (although in this case, as the site already extensively uses px units, it's OK to use them on this project)
2. This might have unintended consequences for other column layouts, e.g. having 2 `col-6` side by side will end up larger than a `col-12`, because `col-6` is not set to the same max-width. This might be intended, but just wanted to flag this incase it's not been considered.